README.md

```diff
@@ -54,6 +54,16 @@ The key features of Lance include:
 pip install pylance
 ```
 
+To install a preview release:
+
+```shell
+pip install --pre --extra-index-url https://pypi.fury.io/lancedb/ pylance
+```
+
+> [!TIP]
+> Preview releases are released more often than full releases and contain the
+> latest features and bug fixes. They receive the same level of testing as full releases.
```

Comment:
So what determines, for us, when we should create a stable release again?  Is it just some rough calendar schedule?

Reply:
I think there's three general triggers:

1. There's a bugfix we urgently want to get out to a broad audience
2. We want to make a release of LanceDB that requires new features from Lance (LanceDB can't depend on a preview release of Lance)
3. It's been two weeks since we last released a full release.